src/mgos_rpc.c

```diff
@@ -154,17 +156,25 @@ static void mgos_rpc_http_handler(struct mg_connection *nc, int ev,
 #endif /* defined(MGOS_HAVE_HTTP_SERVER) && MGOS_ENABLE_RPC_CHANNEL_HTTP */
 
 #if MGOS_ENABLE_SYS_SERVICE
+static void mgos_sys_reboot_abort_tmr(void *opaque) {
+  abort();
```

Comment:
add a log info line just before announcing that abort is intentional.

Reply:
Done; however not here but in the RPC handler: that way the logging is in the same manner as that for rebooting normally.

Reply:
i think it's nice to also have a log line just before the abort itself, like we have for reboot here - https://github.com/cesanta/mongoose-os/blob/master/src/mgos_system.c#L43-L44